### PR TITLE
Feature/xbox connected

### PR DESCRIPTION
**Pull request by toragunstad**
Adds functionality that shows when Xbox is connected or disconnected

- PopUp message in bottom-center of video window for 5 seconds
- Re-written GamePadWrapper to functional component (to be able to use hook useAlert)

Haven't tested that everything works smooth with simulator, but seems to work fine when connecting/disconnecting 😇

### CODE REVIEW

File: src/VideoComponents/GamepadWrapper.jsx [hunk 1]
```diff
@@ -1,51 +1,54 @@
...
     console.log(`Gamepad ${gamepadIndex} connected!`);
-  }
+    alert.success('XBox Controller is connected!');
+  };
 
-  disconnectHandler(gamepadIndex) {
+  const disconnectHandler = gamepadIndex => {
     console.log(`Gamepad ${gamepadIndex} disconnected!`);
-  }
+    alert.error('Note! XBox Controller disconnected');
```
Comment (Trebua) [inline src/VideoComponents/GamepadWrapper.jsx]: maybe rephrase to "Xbox controller disconnected"

File: src/VideoComponents/GamepadWrapper.jsx [hunk 2]
```diff
@@ -1,51 +1,54 @@
...
 
-class GamepadWrapper extends Component {
-  connectHandler(gamepadIndex) {
+function GamepadWrapper() {
+  const alert = useAlert();
+
+  const connectHandler = gamepadIndex => {
     console.log(`Gamepad ${gamepadIndex} connected!`);
-  }
+    alert.success('XBox Controller is connected!');
```
Comment (Trebua) [inline src/VideoComponents/GamepadWrapper.jsx]: Rephrase to 'Xbox Controller is connected'

File: src/index.jsx [hunk 3]
```diff
@@ -3,7 +3,22 @@
...
+import { positions, Provider } from 'react-alert';
+import AlertTemplate from 'react-alert-template-basic';
 import ViewManager from './ViewManager';
 import './index.css';
 
-ReactDOM.render(<ViewManager />, document.getElementById('root'));
+// place pop-up messages in bottom center of screen for 5s
+const options = {
+  timeout: 5000,
+  position: positions.BOTTOM_CENTER,
```
Comment (Trebua) [inline src/index.jsx]: Is it possible to get it in the bottom left corner instead? For me it is at the bottom and like 30px to the left

**Comment by SimenSverdrup** [comment]
Looks great!
